### PR TITLE
Add more explanations about how to add information to the website

### PR DESCRIPTION
**Pull request by hegner**

### CODE REVIEW

**Comment by jouvin** [comment]
As a general remark, try to respect casing for product/service names. For example, `github`  should be written `GitHub`. A global replace is needed basically: I don't add it as a comment to each instance!

File: get_involved.md [hunk 1]
```diff
@@ -6,30 +6,48 @@ layout: default
 
 # How to get involved
 
-Our goal is to make every effort going on as transparent and open as possible. Our official communication channel is the hep-sf-forum. Simply send a mail with ‘subscribe’ as the subject (not content) to hep-sf-forum+subscribe@googlegroups.com
+Our goal is to make every effort going on as transparent and open as possible.
+Our official communication channel is the hep-sf-forum. Simply send a mail with
+‘subscribe’ as the subject (not content) to
```
Comment (jouvin) [inline get_involved.md]: `not content`-> `no content` (not that the subject itself is useless...).

File: get_involved.md [hunk 2]
```diff
@@ -6,30 +6,48 @@ layout: default
...
+hep-sf-forum+subscribe@googlegroups.com
 
-In addition, there is a HEP S&C community mailing list for occasional HEP software and computing community mailings. Everyone involved or interested in HEP S&C is encouraged to sign up to this list. It will be used for occasional mailings of community wide interest, such as announcements of HEP S&C conferences, workshops and schools.
+In addition, there is a HEP S&C community mailing list for occasional HEP
+software and computing community mailings. Everyone involved or interested in
```
Comment (jouvin) [inline get_involved.md]: I'd say: `...community mailings. It is not strictly related to HSF: everyone involved...`

File: get_involved.md [hunk 3]
```diff
@@ -6,30 +6,48 @@ layout: default
...
+no CERN account needed. So is Vidyo, the conference system we use. Connection
+info is on the indico pages. However should you need to upload material to the
+agenda, a CERN account is needed. CERN provides lightweight accounts to non CERN
+affiliated people when needed. See https://account.cern.ch/account/Externals/
 
-# Giving input to the HSF
+## Giving input to the HSF Startup Team
 
-In this document we provide background information, guidelines and a questionnaire for giving input to the HSF, and as background to discussions with the HSF startup team. We'd love to hear from you. If you'd like to arrange a discussion, contact any member of the startup team or email the startup team at hep-sf-startup-team@googlegroups.com.
+If the existing [fora](/forums.html) do not seem to be the right place to ask
```
Comment (jouvin) [inline get_involved.md]: I'd say: `If you are not confortable with raising an issue in the [forums](/forums.html] or if you are afraid that this is not the right place for it, feel free...`

**Comment by jouvin** [comment]
`The same remark about casing applies to `indico`. Should be `Indico`.

File: howto.md [hunk 4]
```diff
@@ -14,11 +14,16 @@ This website is implemented using [github's Pages service](https://pages.github.
 
 ## How to add and edit information
 
-To create or edit files you have to use the [github repository of the HSF website](https://github.com/HEP-SF/hep-sf.github.io/), so you need to be an [HSF github repository](https://github.com/HEP-SF) user. Talk to any member of the startup team.
+For adding information to this page or improving it, we follow the *[pull request](https://help.github.com/articles/using-pull-requests/)* workflow in GitHub.
 
-If you wish (and it is recommended) you can easily set up a local instance of the newsletter site in order to preview submissions. See the [documentation on installing and running Jekyll](https://help.github.com/articles/using-jekyll-with-pages/). The website uses user pages, ie use the master branch.
+Just fork our [HSF website repository](https://github.com/HEP-SF/hep-sf.github.io), edit the
+files you want to edit and open a pull request.
```
Comment (jouvin) [inline howto.md]: Before opening a pull request, one must push back to its fork...

File: howto.md [hunk 5]
```diff
@@ -14,11 +14,16 @@ This website is implemented using [github's Pages service](https://pages.github.
...
 
-If you only do simple operations like the ones mentioned below, the GitHub interface itself gives you all options to add and edit such files in the browser.
+If you wish (and it is recommended) you can easily set up a local instance of the newsletter site in order to preview your submissions. See the [documentation on installing and running Jekyll](https://help.github.com/articles/using-jekyll-with-pages/). The website uses user pages, ie use the master branch.
```
Comment (jouvin) [inline howto.md]: * `[documentation on installing and running Jekyll]` -> `[documentation] on installing and running Jekyll`
* 'The website uses user pages, ie use the master branch' -> 'The website uses the `master` branch of the `hep-sf.github.io` repository'

File: get_involved.md (hunk 1 above)
Comment (hegner) [inline get_involved.md]: agreed.

File: get_involved.md (hunk 2 above)
Comment (hegner) [inline get_involved.md]: agreed.

File: get_involved.md (hunk 3 above)
Comment (hegner) [inline get_involved.md]: sounds nicer, yes.

File: howto.md (hunk 4 above)
Comment (hegner) [inline howto.md]: thanks. makes more sense now.

**Comment by hegner** [comment]
I incorporated the proposals and fixed the GitHub / Indico naming.

File: howto.md [hunk 6]
```diff
@@ -14,11 +14,16 @@ This website is implemented using [github's Pages service](https://pages.github.
...
+
+If you only do simple operations like the ones mentioned below, the GitHub
+interface itself gives you all options to add and edit such files in the browser.
+So no local checkout is required.
```
Comment (jouvin) [inline howto.md]: I'd say: `Despite this is not the recommended option, if you are not confortable with Git and you only want to do simple changes, the GitHub web interface allows you to add and edit the files in your browser. In this case, you don't need to have a local clone of your personal fork`on GitHub.`

File: howto.md [hunk 7]
```diff
@@ -6,19 +6,24 @@ layout: default
...
-If you wish (and it is recommended) you can easily set up a local instance of the newsletter site in order to preview submissions. See the [documentation on installing and running Jekyll](https://help.github.com/articles/using-jekyll-with-pages/). The website uses user pages, ie use the master branch.
+Just fork our [HSF website repository](https://github.com/HEP-SF/hep-sf.github.io), edit the
+files you want to edit, push them to your fork, and open a pull request.
 
-If you only do simple operations like the ones mentioned below, the GitHub interface itself gives you all options to add and edit such files in the browser.
+If you wish (and it is recommended) you can easily set up a local instance of the newsletter site in order to preview your submissions. See the [documentation on installing and running Jekyll](https://help.github.com/articles/using-jekyll-with-pages/).
+The website uses the master branch of the hep-sf.github.io repository.
+If you only do simple operations like the ones mentioned below, the GitHub
+interface itself gives you all options to add and edit such files in the browser.
+So no local checkout is required.
```
Comment (jouvin) [inline howto.md]: I'd say: `Despite this is not the recommended option, if you are not confortable with Git and you only want to do simple changes, the GitHub web interface allows you to add and edit the files in your browser. In this case, you don't need to have a local clone of your personal fork`on GitHub.`

File: howto.md [hunk 8]
```diff
@@ -6,19 +6,24 @@ layout: default
...
+For adding information to this page or improving it, we follow the *[pull request](https://help.github.com/articles/using-pull-requests/)* workflow in GitHub.
 
-If you wish (and it is recommended) you can easily set up a local instance of the newsletter site in order to preview submissions. See the [documentation on installing and running Jekyll](https://help.github.com/articles/using-jekyll-with-pages/). The website uses user pages, ie use the master branch.
+Just fork our [HSF website repository](https://github.com/HEP-SF/hep-sf.github.io), edit the
+files you want to edit, push them to your fork, and open a pull request.
 
-If you only do simple operations like the ones mentioned below, the GitHub interface itself gives you all options to add and edit such files in the browser.
+If you wish (and it is recommended) you can easily set up a local instance of the newsletter site in order to preview your submissions. See the [documentation on installing and running Jekyll](https://help.github.com/articles/using-jekyll-with-pages/).
```
Comment (jouvin) [inline howto.md]: `[documentation on installing and running Jekyll]]` -> `[documentation] on installing and running Jekyll`

Comment (hegner) [inline howto.md]: agreed.

**Comment by jouvin** [comment]
Also, in the `Adding to the newsletter` section, the directory mentioned is incorrect: `workinggroups` -> `newsletter`

File: howto.md (hunk 7 above)
Comment (hegner) [inline howto.md]: will be changed in next update.

**Comment by hegner** [comment]
Let me know once you are done w/ the review. I'll wait with my next push till then.

**Comment by jouvin** [comment]
In the first sentence in the page, I'd replace ` HSF GitHub repository people` by `HSF GitHub contributors` (making only `contributors` a link).

**Comment by jouvin** [comment]
I'd also replace: `We all like to work in code editors; this lets you write content in a friendly editor using the easy markdown syntax (which is used by GitHub itself).` by `The site content is written using the Markdown syntax.`

(`markdown`  is another word that must be capitalized!).

**Comment by jouvin** [comment]
I think I'm done for now! Thanks again for the work done!

**Comment by jouvin** [comment]
One more thing! `front-matter` is mentioned several times. It'd be good in `how to add and edit` to add a few word on the fact that all files starts with a few special lines delimited by `---` lines than contain metadata about the page and are called `front-matter`.

**Comment by jouvin** [comment]
And one last thing: the page title in the front-matter is incorrect: `newsletter` shoud be removed.

**Comment by hegner** [comment]
Incorporated your additional comments; should we merge?

**Comment by jouvin** [comment]
It looks much better! I have still a few (minor remark):
* ` GitHub’s Pages service`: service should not be part of the anchor (in general we tend to put to many words in the anchor, IMO).
* `Pages uses Jekyll` link: `Pages` and `Jekyll` should be 2 separate links
* ` HSF website repository`: `HSF` should not be part of the anchor
* `liquid` in `side menu customization` should be a link
* `The liquid template engine used by Jekyll`: only `liquid` should be a link
* `liquid` should be `Liquid`
* `markdown` should be `Markdown` (at least one occurence left at the end)
* `Markdown syntax`: only `Markdown` should be in the anchor (2 occurences at least)

Should be perfect then!

**Comment by jouvin** [comment]
And as usual one more thing! I'd renamed the file `howto` to `howto-contribute` in order to avoid future possible conflicts (also requires editing reference to it).

**Comment by hegner** [comment]
Can you make the last point a bug report?

**Comment by jouvin** [comment]
Why did you close the PR?

**Comment by hegner** [comment]
Did I close? By accident then.

**Comment by jouvin** [comment]
One last detail that you missed in my previous comment: the first reference to `Liquid` is not a link and should be one IMO. Then I am happy to merge!

**Comment by hegner** [comment]
Done

**Comment by jouvin** [comment]
Rename request tracked as a separate issue (#5), as requested by @hegner .

**Comment by jouvin** [comment]
Thanks for all the refinements done! Merging...

**Comment by hegner** [comment]
Thanks.

**Pull request merged**